lib/logthrdestdrv.c

```diff
@@ -804,6 +850,17 @@ log_threaded_dest_driver_set_max_retries(LogDriver *s, gint max_retries)
   self->retries_max = max_retries;
 }
 
+LogThreadedDestWorker *
+_lookup_worker(LogThreadedDestDriver *self, LogMessage *msg)
```

Comment:
I would call select_worker or pick_worker or similar. As we have an array of workers, maybe we need a real lookup method later.

Reply:
Agree

Reply:
I would change this name in a followup branch, not this one, as it already received some feedback and I have stacks of followup work on top of this.

But please call me on this should I not address that in the load balancing patches.

Thanks.